### PR TITLE
[WIP] Style all the error messages

### PR DESCRIPTION
**Pull request by briandant**
*Note: This includes the reloading commit, which is not on develop yet and will be handled in [another PR](https://github.com/appsembler/xblock-launchcontainer/pull/12/files).*

launcherrequesterrorlist
launcherrequestsuccess
launcherrequesterror
launcherrequestpending

https://trello.com/c/gsTtWI9c/1135-2-make-the-error-messages-better-upon-a-400

### CODE REVIEW

File: launchcontainer/static/html/launchcontainer.html [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-    <input type="text" class="hide" id="launcher_project" name="project" value="{{ project }}" />
-    <input type="text" class="hide" id="launcher_token" name="token" value="{{ project_token }}" />
-    <button type="submit" class="ui-priority-primary" id="launcher_submit">Click to launch your {{ project_friendly }} lab</button>
-  </form>
+  <div class="launcher-form--container">
+    <form id="launcher_form">
+      <input type="text" class="hide" id="launcher_email" name="owner_email" {% if user_email %}value="{{ user_email }}"{% endif %} placeholder="Please enter your email address" />
+      <button type="submit" class="ui-priority-primary" id="launcher_submit">Click to launch your {{ project_friendly }} lab</button>
+    </form>
+  </div>
```
Comment (briandant) [inline launchcontainer/static/html/launchcontainer.html]: Fields are passed in below in the JS, so these form fields are not needed (I kept the email there b/c the xblock-sdk uses it).

File: launchcontainer/static/js/src/launchcontainer.js [hunk 2]
```diff
@@ -10,15 +10,29 @@ function LaunchContainerXBlock(runtime, element) {
...
+                                + "<span class='icon alert-icon fa fa-"+$iconMap[$type]+"'></span>"
+                                  + "<div class='launch-alert--message'>"
+                                    + "<h3>"+$title+"</h3>"
+                                    + "<p>" 
+                                      + $message
+                                    + "</p>" 
+                                  + "</div>"
+                                + "</div>"
+        return $renderedTemplate
+      }
```
Comment (briandant) [inline launchcontainer/static/js/src/launchcontainer.js]: This above is just a little template function for now. We should probably get mustache or something, but I was having trouble loading it in the XBlock world. Not sure if it's possible.

File: launchcontainer/static/js/src/launchcontainer_edit.js [hunk 3]
```diff
@@ -1,51 +1,35 @@
 function LaunchContainerEditBlock(runtime, element) {
```
Comment (briandant) [inline launchcontainer/static/js/src/launchcontainer_edit.js]: This is all handled in [the other PR](https://github.com/appsembler/xblock-launchcontainer/pull/12/files). No changes here.

**Review by tomaszzielinski — COMMENTED** [review]
This looks good except that I'm concerned about manipulating DOM using strings instead of using safe DOM methods (or even jQuery)--even if we're good right now any change in the future might result in XSS vulnerability...

**Comment by briandant** [comment]
@tomaszzielinski: You're exposing my JavaScript weaknesses—good work! I think I'll turn that into some jQuery and see what you think. 

Many thanks for these reviews 👍 🇵🇱

**Comment by OmarIthawi** [comment]
@briandant Just a reminder about this PR, since the reload PR (https://github.com/appsembler/xblock-launchcontainer/pull/12) has been merged.

**Comment by briandant** [comment]
Thanks for the reminder, @OmarIthawi. I've added this to our current list of TODOs.

**Pull request closed without merging**